### PR TITLE
APR-1363 - QnA styling

### PR DESCRIPTION
**Pull request by gdbarnes**
This branch should be deployed and tested alongside the branch in the `das-apply-service` repository named:
`APR-1363_styling`

### CODE REVIEW

File: src/SFA.DAS.QnA.Database/projects/roatp/sections/section1.2.json [hunk 1]
```diff
@@ -128,13 +128,13 @@
...
                     "QuestionTag": null,
-                    "Label": "<span class='govuk-label'>Company name</span>",
+                    "Label": "Company name",
                     "ShortLabel": "",
                     "QuestionBodyText": "",
                     "Hint": "",
                     "Input": {
                         "Type": "Text",
-                        "InputClasses": "app-uppercase",
+                        "InputClasses": "govuk-!-width-two-thirds",
```
Comment (David-B-Read) [inline src/SFA.DAS.QnA.Database/projects/roatp/sections/section1.2.json:137]: app-uppercase missing?

File: src/SFA.DAS.QnA.Api.Views/Views/QuestionPartials/_CheckboxList.cshtml [hunk 2]
```diff
@@ -55,7 +55,8 @@
 </div>
 
 <script type="text/javascript">
-    if (document.querySelector('.js-check-all-container')) {
+    // Is ".js-check-all-container" shown on the page
+    if (document.querySelector('.js-check-all-container').offsetParent) {
```
Comment (MarkFCain) [inline src/SFA.DAS.QnA.Api.Views/Views/QuestionPartials/_CheckboxList.cshtml:59]: This is definitely ok, is it? Lot of pages will be affected by this, but if you're sure... 💃

Comment (gdbarnes) [reply src/SFA.DAS.QnA.Api.Views/Views/QuestionPartials/_CheckboxList.cshtml:59]: Yes. This prevents the indent which is making all the checkboxes look incorrectly aligned in apply.

File: src/SFA.DAS.QnA.Database/projects/roatp/sections/section1.2.json (hunk 1 above)
Comment (gdbarnes) [reply src/SFA.DAS.QnA.Database/projects/roatp/sections/section1.2.json:137]: We don't want to force them to put their company name in block capitals do we?

Comment (David-B-Read) [reply src/SFA.DAS.QnA.Database/projects/roatp/sections/section1.2.json:137]: Yes, that was the requirement on the original story. It's returned in upper case by the UKRLP and Companies House services

Comment (gdbarnes) [reply src/SFA.DAS.QnA.Database/projects/roatp/sections/section1.2.json]: Okay. I've put it back in 👌

**Pull request merged**